src/runtime_src/core/tools/xbmgmt2/OO_Config.cpp

```diff
@@ -256,6 +256,7 @@ OO_Config::OO_Config( const std::string &_longName, bool _isHidden)
   m_optionsDescription.add_options()
     ("device,d", boost::program_options::value<decltype(m_devices)>(&m_devices)->multitoken(), "The Bus:Device.Function (e.g., 0000:d8:00.0) device of interest")
     ("retention", boost::program_options::value<decltype(m_retention)>(&m_retention),"Enables / Disables memory retention.  Valid values are: [ENABLE | DISABLE]")
+    ("ddr", boost::program_options::bool_switch(&m_ddr), "Memory type to be enabled/disabled for retention")
```

Comment:
Why is this option no longer hidden?  Comments didn't see to indicate what this was being done.

If it is needed, then the describing needs to be updated.  For example:
`Enable DDR memory for retention`

Reply:
This is a new feature that was added as a hidden command since it was a new feature. But over time it has matured and ready to be exposed to the users. We exposed it to the users in xbmgmt, but forgot to do so in xbmgmt2